src/Pages/SettingsPage/style.module.scss

```diff
@@ -0,0 +1,156 @@
+@import "../../colors.module.css";
+@import "../../fonts.module.css";
+
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 4rem 0;
+  width: 55%;
+
+  @media (max-width: 768px) {
+    width: 100%;
+  }
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.profilePicWrapper {
+  width: 100%;
+
+  .profilePic {
+    width: 6rem;
+    height: 6rem;
+    border: 4px solid var(--NAVY_BLUE);
+    border-radius: 50%;
+    padding: 6px;
+  }
+}
+
+.profilePicForm {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 1em;
+
+  .selectedFileName {
+    font-style: italic;
+    color: var(--DARK_GRAY);
+    margin-top: 0.5em;
+    font-size: 1rem;
+  }
+}
+
+.profilePicForm input[type="file"] {
+  display: none;
+}
+
+.profilePicButton {
+  width: 8rem;
+  background-color: var(--BANANA_YELLOW);
+  color: var(--NAVY_BLUE);
+  border: none;
+  margin: 0.25rem;
+  padding: 0.45rem 0.85rem;
+  border-radius: 5px;
+  font-size: 1rem;
+  font-weight: bold;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+
+  &:hover {
+    background-color: var(--FADED_NAVY_BLUE);
+    cursor: pointer;
+  }
+}
+
+.fileUploadButton {
+  width: 8rem;
+  background-color: var(--LIGHT_GRAY);
+  color: var(--NAVY_BLUE);
+  margin: 0.25rem;
+  padding: 0.45rem 0.85rem;
+  border-radius: 5px;
+  font-weight: bold;
+  cursor: pointer;
+  border: none;
+  display: inline-block;
+}
+
+.fileUploadButton:hover {
+  cursor: pointer;
+  background-color: var(--FADED_NAVY_BLUE);
+}
+
+.editPassword {
+  font-family: Open Sans;
+  font-style: normal;
+  font-weight: bold;
+  color: var(--NAVY_BLUE);
+  text-decoration: none;
+}
+
+.nameHeader {
+  font-family: Open Sans;
+  font-style: normal;
+  color: var(--NAVY_BLUE);
+  letter-spacing: 0.04em;
+  font-weight: bold;
+  margin-bottom: 0.8em;
+}
+
+.passwordContainer {
+  margin-top: 2em;
+  display: flex;
+  justify-content: space-between;
+}
+
+.editLink {
+  font-family: Open Sans;
```

Comment:
I'm pretty sure this just came over from the previous css file, but does Open Sans need to be in quotes? Also, it seems like we have the `font-family` sprinkled across a ton of stylesheets. Can we just set this globally so it's not necessary to set everywhere individually?

Reply:
@tackc makes total sense 😆 I also noticed that each page has a `container` wrapper that we could prob move to a single parent